spec/lib/rmt/cli/repos_spec.rb

```diff
@@ -185,7 +184,7 @@
           let(:argv) { [command_name] }
           let(:expected_output) do
             Terminal::Table.new(
-              headings: ['SCC ID', 'Name', 'Description', 'Mandatory?', 'Mirror?', 'Last mirrored'],
+              headings: ['SCC ID', 'Product', 'Mandatory?', 'Mirror?', 'Last mirrored'],
```

Comment:
Don't we have to make similar adjustments to `products_spec`? I see the tests passing though. :thinking:

Reply:
Updated the `products_spec` in my latest commit.